### ISSUE TITLE
MBS Direct Mobile Site

### ISSUE THREAD

**Issue by 34code** [bsgreenb/Open-Textbooks#7]
I do not seem to have any luck finding the mobile sites for simplifying scraping from MBS Direct sites.. 

According to the documentation, http://bookstoreurl.com/mobile/textbooks.aspx should give me the mobile version...

However, for the following bookstores... I cannot seem to find a mobile version:

http://bookstore.mbsdirect.net/cdu.htm
http://bookstore.mbsdirect.net/bsu.htm
http://bookstore.mbsdirect.net/depaul.htm

Not having MBS Direct is not a dealbreaker as most institutions are leaving / have left it due to its complicated user interface... a search for "university" at http://www.mbsdirect.net/index.php will highlight what im talking about..

That said, it would be nice if I could find the mobile site in order to scrape MBS Direct.

**Comment by bsgreenb** [comment]
Interesting, this looks to be a new version of MBS, or at least a different
version than what people have when they host MB on their own servers.  Are
those the only 3 MBS that you're stuck with this version for?

Either way, if you want to do these ones it looks like you'll need to write
a new scraper. It shouldn't be too hard since there's only a few steps
required to getting data.  If you take on that task, let me know if you
need any help with that.

**Comment by 34code** [comment]
Actually this seems to be the issue for every hosted MBS which is part of the http://bookstore.mbsdirect.net url... its not complicated to write a scraper for this.. and I'll do that soon but its not priority at them moment.. I should have a working version in a few months' time..

**Comment by bsgreenb** [comment]
Thanks Sambit.  If you could file a pull request to Open-Textbooks when you
write and successfully test that code that'd be great.

**Comment by 34code** [comment]
Sure. Will do that... Also, I wanted to ask if you received my email I sent to ben@textyard.com?

**Comment by bsgreenb** [comment]
Sounds good.

I responded to your email a while ago, did you not get a response in your
inbox?

**Comment by 34code** [comment]
nope.. didnt seem to have received it.. maybe its a problem at your mail server's end? or maybe its my end... is possible to try sending it again.. if you cc 34sambit@gmail.com, that'll make sure I receive it..Thanks.

**Comment by bsgreenb** [comment]
Sent it to sambit@botflip.com originally. forwarded now.
